platform-configs/include/novacustom-mtl.robot

```diff
@@ -0,0 +1,30 @@
+*** Variables ***
+# Flash
+${FLASH_SIZE}=                      ${16*1024*1024}
+
+# CPU
+${INITIAL_CPU_FREQUENCY}=           2800
+${DEF_CORES_PER_SOCKET}=            4
+${DEF_THREADS_PER_CORE}=            2
```

Comment:
This variable doesn't work with heterogeneous CPUs like meteorlake, but that's a problem for later.